modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -4,8 +4,10 @@
 import { Injectable, Optional } from '@angular/core';
 import { HttpClient, HttpHeaders, HttpParams, HttpParameterCodec } from '@angular/common/http';
 import { CustomHttpParameterCodec } from '../encoder';
-import { of, throwError } from 'rxjs';
-import { mergeMap } from 'rxjs/operators';
+// prettier-ignore
```

Comment:
Why is this necessary now?

Reply:
Because the `throwError` is not always used in every api service. It is only used for operations (service functions) that contain parameters. That is not easily to check for the imports so I decided to simply ignore it.